src/flows/pipes/Notification/NotificationMonacoEditor.tsx

```diff
@@ -59,7 +60,10 @@ const NotificationMonacoEditor: FC<Props> = ({
           scrollBeyondLastLine: false,
           readOnly: !!readOnly,
         }}
-        editorDidMount={editorDidMount}
+        editorDidMount={ed => {
+          pointToCorrectWorkers()
+          editorDidMount(ed)
```

Comment:
Always reference the same editor worker instance:
    * can get multiple instances with ESM loading (webpack plugin)
    * plus we have 2 stages of webpack builds (`webpack.vendor.ts`, and `webpack.common.ts`) --> but at runtime want to have only 1 instance.
    * simple solution: both resolve to `filename: '[name].worker.js'` on build
    * HERE it's referencing the correct, single instance workers